nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -1064,6 +1074,26 @@ def predict_step(self, batch, batch_idx=0, dataloader_idx=0, has_processed_signa
     def adapter_module_names(self) -> List[str]:
         return ['', 'encoder', 'transf_encoder', 'transf_decoder']
 
+    @property
+    def oomptimizer_schema(self) -> list[dict]:
+        """
+        Return a typing schema for optimal batch size calibration for various
+        sequence lengths using OOMptimizer.
+        """
+        assert hasattr(self, "tokenizer"), "OOMptimizer currently supports only models that use tokenizers."
+        return [
+            {"type": NeuralType(("B", "T"), AudioSignal()), "seq_length": "input"},
+            {"type": NeuralType(("B",), LengthsType()), "seq_length": "input"},
+            {
+                "type": NeuralType(("B", "T"), LabelsType()),
+                "seq_length": "output",
+                "vocab_size": self.tokenizer.vocab_size,
+            },
+            {"type": NeuralType(("B",), LengthsType()), "seq_length": "output"},
+            {"type": "dummy"},
```

Comment:
Are these two lines still needed? It looks like "dummy" entries get filtered out. But I may be missing code in lhotse that might work with this.

Reply:
Yes, dummy will pass an empty tensor, but it still needs to be there to match the training_step function signature.